db/schema.rb

```diff
@@ -344,12 +344,12 @@
     t.index ["pipeline_run_id", "tax_id", "count_type", "tax_level"], name: "index_pr_tax_hit_level_tc", unique: true
   end
 
-  create_table "taxon_descriptions", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "taxon_descriptions", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```

Comment:
I believe this change is correct... i wonder how it regressed. 
Can you confirm with @yunfangjuan to be sure?

Reply:
whoops didn't mean to include this. will remove.